cypress/local-only/tests/integration/caseDetail/caseInformation/edit-case-detail.cy.ts

```diff
@@ -0,0 +1,327 @@
+            statisticId: 'ab557361-50ee-4440-aaff-0a9f1bfa30ed',
+            updatedAt: '',
+          },
+        ],
+      },
+      {
+        _sortOrder: 2,
+        docketNumber,
+        lastDateOfPeriod: undefined,
+        statisticId: 'cb557361-50ee-4440-aaff-0a9f1bfa30ed',
+        year: 2019,
+        yearOrPeriod: 'Year',
+        updatedAt: '',
+        irsDeficiencyAmount: '5678',
+        determinationDeficiencyAmount: '300',
+        irsTotalPenalties: '3',
+        determinationTotalPenalties: '2',
+        penalties: [
+          {
+            name: 'John of Salisbury',
+            penaltyAmount: '3', // IRS Notice
+            penaltyId: 'e0a52030-1ba5-4b1a-bb62-b5e591ea434e',
+            penaltyType: PENALTY_TYPES.IRS_PENALTY_AMOUNT,
+            statisticId: 'cb557361-50ee-4440-aaff-0a9f1bfa30ed',
+            updatedAt: '',
+          },
+          {
+            name: 'Roscellinus',
+            penaltyAmount: '2', // Determination
+            penaltyId: 'e0a52030-1ba5-4b1a-bb62-b5e591ea434e',
+            penaltyType: PENALTY_TYPES.DETERMINATION_PENALTY_AMOUNT,
+            statisticId: 'cb557361-50ee-4440-aaff-0a9f1bfa30ed',
+            updatedAt: '',
+          },
+        ],
+      },
+    ];
+  };
+
+  describe('should change the things', () => {
+    before(function () {
+      createAndServePaperPetition({ yearReceived: '1950' }).then(caseRecord => {
+        cy.wrap(caseRecord.docketNumber).as('docketNumber');
+      });
+    });
+
+    it('should change the title of the case', function () {
+      loginAsDocketClerk();
+      cy.visit(`/case-detail/${this.docketNumber}`);
+      getCaseDetailTab('case-information').click();
+      getEditCaseCaptionButton().click();
+      getCaptionTextArea().clear().type('there is no cow level');
+      getButton('Save').click();
+      getCaseTitleContaining(
+        'there is no cow level v. Commissioner of Internal Revenue, Respondent',
+      ).should('exist');
+    });
+
+    it('should change type of case (to deficiency)', function () {
+      getCaseDetailTab('case-information').click();
+      cy.get('[data-testid="edit-case-details-button"]').click();
+      cy.get('[data-testid="case-type-select"]').select(
+        CASE_TYPES_MAP.deficiency,
+      );
+      getButton('Save').click();
+      cy.contains('p', CASE_TYPES_MAP.deficiency);
+    });
+
+    it('should add deficiency statistics', function () {
+      getButton('Statistics').click();
+      for (const statistic of getDeficiencyStatistics(docketNumber)) {
+        cy.contains('a', 'Add New Year/Period').click();
+        cy.get(`[data-testid="year-or-period-${statistic.yearOrPeriod}"]`);
+
+        if (statistic.yearOrPeriod === 'Period') {
+          cy.get('[data-testid="year-or-period-Period"]').click({
+            force: true,
+          });
+          // Get data-testid to work ... using id because we multiply the data-testid apparently
```

Comment:
Well now I'm curious about this id multiplication

Reply:
Ack, thank you for the reminder on this comment. I'm not sure why this was happening, but trying to select by data-testid gave me a "multiple things have this" error.

Reply:
I think it is USDWS funkiness with DateSelector wrapping our input:

<img width="1382" alt="Screenshot 2025-03-19 at 5 46 02 PM" src="https://github.com/user-attachments/assets/59543469-7570-4fe8-ac2c-b50e7c455ca9" />

Reply:
I should have guessed...